build/configs/common_download.sh

```diff
@@ -202,8 +202,14 @@ function get_configured_partitions()
 	else
 		configured_parts=${CONFIG_FLASH_PART_NAME}
 	fi
+	
+	echo $configured_parts
+}
+
+function get_configured_second_partitions()
+{
 	if [[ -n ${CONFIG_ARCH_BOARD_HAVE_SECOND_FLASH} ]];then
-		configured_parts+=${CONFIG_SECOND_FLASH_PART_NAME}
+		configured_parts=${CONFIG_SECOND_FLASH_PART_NAME}
```

Comment:
You add the config in build/configs/rtl8730e/loadable_ext/defconfig only.
Is this working well with others which do not (will not) have the config?

Reply:
I change this.